dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScopeTest.kt

```diff
@@ -648,12 +648,16 @@ internal class RumViewScopeTest {
     }
 
     @Test
-    fun `𝕄 send event 𝕎 handleEvent(ApplicationStarted) on active view`(
+    fun `𝕄 send event with global attributes 𝕎 handleEvent(ApplicationStarted) on active view`(
         @StringForgery(StringForgeryType.ALPHABETICAL) key: String,
         @StringForgery(StringForgeryType.ALPHABETICAL) name: String,
-        @LongForgery(0) duration: Long
+        @LongForgery(0) duration: Long,
+        forge: Forge
     ) {
         // Given
+        GlobalRum.globalAttributes.clear()
```

Comment:
Actually, maybe we should clear those in the `tear down` method to ensure it will behave the same for all tests using it.

Reply:
It is already the case there, here was just an extra safety measure to make if the test suite starts with this method to make sure we don't have left others from other test suites.